deployer/src/deployment/queue.rs

```diff
@@ -539,21 +548,4 @@ ff0e55bda1ff01000000000000000000e0079c01ff12a55500280000",
             "barfoo"
         );
     }
-
-    #[tokio::test]
-    async fn get_secrets() {
```

Comment:
Hmm, mind clarifying why we can remove this?

Reply:
Thought it would be more of an integration test at this point, but no. Added it back.